client/src/components/SupportTools/support-api.tsx

```diff
@@ -215,3 +222,38 @@ export const useClearOfflineResults = () => {
       ]),
   })
 }
+
+export const useUndoRoundStart = () => {
+  const undoRoundStart = async ({
+    roundId,
+  }: {
+    electionId: string
```

Comment:
Though the API path doesn't need an election ID, we use election ID to invalidate the relevant query below

Reply:
Nice, I like that approach